.github/workflows/helm.yaml

```diff
@@ -39,6 +43,35 @@ jobs:
           if [[ -n "$changed" ]]; then
             echo "::set-output name=changed::true"
           fi
+      
+      # Force linting/test in debug runs with a change (this action only runs on chart and helm.yaml workflow diffs anyways)
+      - name: Change Kind to deployment
+        run: "sed -i \"s|kind: [^ ]*|kind: Deployment|g\" charts/descheduler/values.yaml"
 
       - name: Run chart-testing (lint)
         run: ct lint --config=.github/ci/ct.yaml --validate-maintainers=false
+      
+      # Need a multi node cluster so descheduler runs until evictions
+      - name: Create multi node Kind cluster
+        run: |
+          cat > kind-config.yaml <<EOF
+          # three node (two workers) cluster config
+          kind: Cluster
+          apiVersion: kind.x-k8s.io/v1alpha4
+          nodes:
+          - role: control-plane
+          - role: worker
+          - role: worker
+          EOF
+          kind create cluster --name kind --config kind-config.yaml --wait 2m
```

Comment:
Can this not reference the [kind_config](https://github.com/kubernetes-sigs/descheduler/blob/master/hack/kind_config.yaml) used by e2e?

Reply:
for topology spread tests, we label the nodes with unique AZ values

Reply:
Great! using it now.